### DISCUSSION TITLE
middle click range

### DISCUSSION THREAD

**Discussion by vincehi** [rxhanson/Multitouch-Community#102] [Ideas]
I would like to be able to adjust the surface of the middle click. Indeed, everyone's hands are different and when I use BTT I can adjust this so I never have a right or left click when I want to do a middle click.
Great application

**Comment by rxhanson** [comment]
Multitouch works a little differently than BTT here. The right click, if configured, will override the middle click performed by Multitouch, and this is why I don't expose these controls in the UI.

You can adjust these values in Multitouch with the following terminal commands, but the width cannot extend further to the right if you use the right side of the Magic Mouse for right clicking.

defaults write com.brassmonkery.Multitouch oneCenterClickMinX -float 0.34
defaults write com.brassmonkery.Multitouch oneCenterClickMaxX -float 0.55

Those are the default values, so you could play around and see if you get something that works better (greater than 0.55 would get overridden by the right click). You need to restart the app after running that terminal command.

There is a macOS default that you could try modifying, but it's not documented so I can't say if it will work for everyone.

defaults write com.apple.AppleMultitouchMouse MouseButtonDivision -int 55

(55 is the default here as well)
